src/averageAges.js

```diff
@@ -37,7 +44,18 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const filterPeople = withChildren === undefined
```

Comment:
Use a more descriptive variable name instead of 'filterPeople'. For example, 'filteredWomen'.